kiwi/xml_description.py

```diff
@@ -108,6 +112,25 @@ def load(self):
             raise KiwiDescriptionInvalid(message)
         return self._parse()
 
+    def _get_validation_details(self):
+        """
+        Run jing process to validate description against the schema
+
+        Jing command provides detailed feedback in case of schema
+        validation failure
+        """
+        try:
+            cmd = Command.run(
+                ['jing', Defaults.get_schema_file(), self.description_xslt_processed.name],
```

Comment:
I think we should have test if jing is installed prior to calling it. We can't require jing in the spec of the kiwi package because it will pull in the entire java stack. Thus an information to the user that kiwi would provide better error messages if jing is installed and from where it can be installed would be good